pkg/codegen/go/gen.go

```diff
@@ -3797,7 +3785,7 @@ func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error
 			importsAndAliases := map[string]string{}
 			pkg.getImports(r, importsAndAliases)
 			importsAndAliases["github.com/pulumi/pulumi/sdk/v3/go/pulumi"] = ""
-
+			importsAndAliases[path.Join(pkg.importBasePath, "internal")] = ""
```

Comment:
It feels like there should be a method for this, but I don't feel strongly about it.
```suggestion
			importUtilities(importsAndAliases)
```